packages/web/components/Patron/Join/PerksGrid.tsx

```diff
@@ -84,272 +94,344 @@ const LeftArrowStyle = {
+          borderRadius={8}
+          minH="8rem"
+          bg="whiteAlpha.200"
+          mb={{ base: 0, md: '1em' }} // the arrow on the next Box arrow makes the margin at base breakpoint
+        >
+          <PerksHeader
+            title={'Silver League'}
+            count={getLeagueCount(PlayerRank_Enum.Silver)}
+            pSeeds={leaguePSeedsByRank[PlayerRank_Enum.Silver]}
+            amountUsd={
+              pSeedPrice
+                ? leaguePSeedsByRank[PlayerRank_Enum.Silver] * pSeedPrice
+                : null
+            }
+          />
 
-        <Box p={8} width="100%" color="white">
-          <Flex width="100%" flexDirection="row" flexWrap="wrap">
-            {GoldenLeagueList.map((text: string) => (
-              <LeagueCardItem text={text} />
-            ))}
-          </Flex>
-        </Box>
-      </VStack>
-    </Box>
+          <Box p={8} width="100%" color="white">
+            <List textAlign="left">
+              <Flex width="100%" flexDirection="row" flexWrap="wrap">
+                {SilverLeagueList.map((text: string, index) => (
+                  <LeagueCardItem key={index} text={text} />
+                ))}
+              </Flex>
+            </List>
+          </Box>
+        </VStack>
+      </Box>
 
-    <Box
-      width={{ base: '100%', md: '50%' }}
-      float={{ base: 'none', md: 'right' }}
-      _before={RightArrowStyle}
-    >
-      <VStack
-        spacing={0}
-        borderRadius={8}
-        minH="5rem"
-        bg="whiteAlpha.200"
-        mb={{ base: 0, md: '1em' }} // the arrow on the next Box arrow makes the margin at base breakpoint
+      <Box
+        width={{ base: '100%', md: '50%' }}
+        float={{ base: 'none', md: 'left' }}
+        _before={LeftArrowStyle}
       >
-        <PerksHeader title={'Platinum League'} count={7} amount={3631} />
+        <VStack
+          spacing={0}
+          borderRadius={8}
+          minH="8rem"
+          bg="whiteAlpha.200"
+          mb={{ base: 0, md: '1em' }} // the arrow on the next Box arrow makes the margin at base breakpoint
+        >
+          <PerksHeader
+            title={'Gold League'}
+            count={getLeagueCount(PlayerRank_Enum.Gold)}
+            pSeeds={leaguePSeedsByRank[PlayerRank_Enum.Gold]}
+            amountUsd={
+              pSeedPrice
+                ? leaguePSeedsByRank[PlayerRank_Enum.Gold] * pSeedPrice
+                : null
+            }
+          />
 
-        <Box p={8} width="100%" color="white">
-          <Flex width="100%" flexDirection="row" flexWrap="wrap">
-            {PlatinumLeagueList.map((text: string) => (
-              <LeagueCardItem text={text} />
-            ))}
-          </Flex>
-        </Box>
-      </VStack>
-    </Box>
+          <Box p={8} width="100%" color="white">
+            <Flex width="100%" flexDirection="row" flexWrap="wrap">
+              {GoldenLeagueList.map((text: string, index) => (
+                <LeagueCardItem key={index} text={text} />
```

Comment:
```suggestion
                <LeagueCardItem key={index} {...{ text }} />
```

Reply:
I find this syntax overall less readable when its only a single prop